remember_remember.py

```diff
@@ -0,0 +1,19 @@
+from PIL import Image
+
+
+def read_photo_message(file_path: str):
+    """
+    This function receive path to image, read the image and print the encrypted message.
+    :param file_path: The path to the image.
+    """
+    image = Image.open(file_path)
+    pixel = image.load()
+    for col in range(image.size[0]):
+        for row in range(image.size[1]):
+            if pixel[col, row] == 1:
+                print(chr(row), end="")
```

Comment:
You should be returning the message and not printing it. Why would you "crack the code" if you can't use it again?